Default.sublime-keymap

```diff
@@ -25,6 +25,24 @@
             }
         ]
     },
+    {
+        "command": "lsp_commit_completion_with_opposite_insert_mode", 
```

Comment:
There is a trailing space here.
Also I wouldn't put this key binding so high in the file. There are more relevant ones that probably deserve more attention.